metrics/web/lib/features/auth/presentation/strings/login_strings.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:metrics/features/auth/presentation/pages/login_page.dart';
+
+/// Holds the strings for the [LoginPage].
+///
+/// Preferably, add the string to this file instead of hardcoding them into UI
+/// to make them available in tests and avoid code duplication.
+class LoginStrings {
+  static const String email = 'Email';
+  static const String password = 'Password';
+  static const String signIn = 'Sign in';
+
+  static const emailIsRequired = 'Email address is required';
```

Comment:
```suggestion
  static const String emailIsRequired = 'Email address is required';
```
It will be better to specify type here to keep the code more consistent.